pkg/packagekit/kpatch.jsx

```diff
@@ -0,0 +1,325 @@
+                        if (!data) {
+                            resolve();
+                        } else {
+                            const autoupdate = data.split("\n").find(l => l.indexOf("autoupdate") >= 0);
+                            const auto = autoupdate.endsWith("True");
+                            this.setState({ auto: auto, futureAuto: auto }, resolve);
+                        }
+                    })
+                    .catch(reject);
+        });
+    }
+
+    handleInstall() {
+        this.setState({ updating: true });
+        install_dialog(this.state.missing)
+                .then(() => {
+                    this.setState({ missing: [], updating: false });
+                })
+                .catch(() => this.setState({ updating: false }));
+    }
+
+    handleChange() {
+        this.setState({ updating: true });
+
+        const promises = [];
+
+        if (this.state.auto !== this.state.futureAuto)
+            promises.push(this.installPatches(this.state.futureAuto));
+
+        if (this.state.enabled !== this.state.futureEnabled)
+            promises.push(this.handleService(this.state.futureEnabled));
+
+        Promise.all(promises)
+                .then(() => this.setState({ showModal: false }))
+                .catch(console.log)
+                .finally(() => {
+                    this.checkSetup();
+                    this.setState({ updating: false });
+                });
+    }
+
+    render() {
+        // Not yet recognized
+        if (this.state.loaded === false)
+            return null;
+
+        // Not supported on this system
+        if (this.state.unavailable.length > 0)
+            return null;
+
+        let state = _("Enabled");
+        let actionText = _("Edit");
+        let action = () => this.setState({ showModal: true });
+
+        if (this.state.missing.length > 0 && this.state.unavailable.length === 0) {
+            state = _("Not installed");
+            actionText = _("Install");
+            action = this.handleInstall;
+        } else if (this.state.auto === false) {
+            state = _("Disabled");
+            actionText = _("Enable");
+        }
+
+        const details = [];
+        if (this.state.enabled !== this.state.futureEnabled) {
+            if (this.state.futureEnabled)
+                details.push(_("kpatch.service will be enabled. After each reboot patches will be applied."));
+            else
+                details.push(_("kpatch.service will be disabled. After reboot no patches will be applied."));
+        }
+        // FIXME - should we uninstall current patches as well?
+        if (this.state.auto !== this.state.futureAuto) {
+            if (this.state.futureAuto)
+                details.push(_("Current kernel patches will be installed. Also when a new kernel is available, patches will be automatically installed as well."));
+            else
+                details.push(_("For future kernel versions patches won't be installed."));
```

Comment:
```suggestion
                details.push(_("Available kernel patches will be applied. Patches for new kernel versions will automatically applied."));
            else
                details.push(_("Patches for future kernel versions will not be applied."));
```

Reply:
"Available kernel patches will be applied. Patches for new kernel versions will automatically applied." - doesn't that miss a verb? "will automatically apply" or "will automatically be applied"?